### ISSUE TITLE
NTRs: microlipophagy and regulation of microlipophagy

### ISSUE THREAD

**Issue by robnash** [geneontology/go-ontology#19869]
Hi,

I would like to request the following terms 'microlipophagy' and 'regulation of microlipophagy' be added to the process ontology. 

Here is what I am thinking regarding placement:

Parentage (as a child term of)

GO:0016237	lysosomal microautophagy
----->GO:0000424	micromitophagy
----->GO:0000426	micropexophagy
----->GO:0034727	piecemeal microautophagy of the nucleus
----->GO:XXXXXXX	microlipophagy
---------->GO:YYYYYYY	regulation of microlipophagy

NTR: microlipophagy

Definition: Microautophagy-mediated direct internalization of lipid droplets into a lysosome-like vacuole during nutrient depletion, such as during the transition to stationary phase or in response to nutrient limitation. Microlipophagy is mediated by the formation of sterol-enriched vacuolar microdomain at sites of engulfment.

Synonyms: μ-lipophagy

References:

Zhang X, Evans TD, Jeong SJ, Razani B. Classical and alternative roles for autophagy in lipid metabolism. Curr Opin Lipidol. 2018 Jun;29(3):203-211. doi: 10.1097/MOL.0000000000000509. PMID: 29601311; PMCID: PMC5930069. REVIEW

Seo AY, et al. (2017) AMPK and vacuole-associated Atg14p orchestrate μ-lipophagy for energy production and long-term survival under glucose starvation. Elife. 2017 Apr 10;6:e21690. doi: 10.7554/eLife.21690. PMID: 28394250; PMCID: PMC5407857.

Wang CW, et al. (2014) A sterol-enriched vacuolar microdomain mediates stationary phase lipophagy in budding yeast. J Cell Biol 206(3):357-66
PMID: 25070953

Oku M, et al. (2017) Evidence for ESCRT- and clathrin-dependent microautophagy. J Cell Biol 216(10):3263-3274
PMID: 28838958

NTR: regulation of microlipophagy

Definition: Any process that modulates the frequency, rate or extent of microlipophagy, the microautophagy-mediated direct internalization of lipid droplets into a lysosome-like vacuole during nutrient depletion.

Synonyms:	regulation of μ-lipophagy

References:
Zhang X, Evans TD, Jeong SJ, Razani B. Classical and alternative roles for autophagy in lipid metabolism. Curr Opin Lipidol. 2018 Jun;29(3):203-211. doi: 10.1097/MOL.0000000000000509. PMID: 29601311; PMCID: PMC5930069.

Seo AY, et al. (2017) AMPK and vacuole-associated Atg14p orchestrate μ-lipophagy for energy production and long-term survival under glucose starvation. Elife. 2017 Apr 10;6:e21690. doi: 10.7554/eLife.21690. PMID: 28394250; PMCID: PMC5407857.

Cheers,
Rob

**Comment by pgaudet** [comment]
@ukemi @marcfeuermann @RLovering 
just checking that you have seen this and that it seems OK (as it seems related to the autophagy work).

**Comment by RLovering** [comment]
sounds good to me

**Comment by marcfeuermann** [comment]
It seems to be well documented, so yes, looks okay for me too. I'm always pleased to hear about a new autophagy type :-)

**Comment by robnash** [comment]
Just noticed a minor typo in the definition. Microdomains should be plural. Here is the corrected def.

NTR: microlipophagy
Definition: Microautophagy-mediated direct internalization of lipid droplets into a lysosome-like vacuole during nutrient depletion, such as during the transition to stationary phase or in response to nutrient limitation. Microlipophagy is mediated by the formation of sterol-enriched vacuolar microdomains at sites of engulfment.

**Comment by pgaudet** [comment]
+[Term]
+id: GO:0140504
+name: microlipophagy
+namespace: biological_process
+def: "Microautophagy-mediated direct internalization of lipid droplets into a lysosome-like vacuole during nutrient depletion, such as during the transition to stationary phase or in response to nutrient limitation. Microlipophagy is mediated by the formation of sterol-enriched vacuolar microdomains at sites of engulfment." [PMID:25070953, PMID:28394250, PMID:28838958, PMID:29601311]
+is_a: GO:0016237 ! lysosomal microautophagy
+property_value: term_tracker_item https://github.com/geneontology/go-ontology/issues/19869 xsd:anyURI
+created_by: pg
+creation_date: 2020-08-14T12:53:43Z
+
+[Term]
+id: GO:0140505
+name: regulation of microlipophagy
+namespace: biological_process
+def: "Any process that modulates the frequency, rate or extent of microlipophagy, the microautophagy-mediated direct internalization of lipid droplets into a lysosome-like vacuole during nutrient depletion." [PMID:28394250, PMID:29601311]
+is_a: GO:0065007 ! biological regulation
+property_value: term_tracker_item https://github.com/geneontology/go-ontology/issues/19869 xsd:anyURI
+created_by: pg
+creation_date: 2020-08-14T12:56:29Z

**Comment by pgaudet** [comment]
As fasr as I know we cannot add greek characters, so I didn't create the synonym. 
@balhoff is this right ?

**Comment by balhoff** [comment]
We could add Greek characters, but we don't, since some folks' parsers may not handle properly.

**Comment by pgaudet** [comment]
Right, thanks for the explanation.

**Comment by deustp01** [comment]
Tangent thought - ChEBI allows them in names (along with other non-ASCII stuff used for chemical nomenclature) and also provides an ASCII version of all such names - e,g, CHEBI:137927
ChEBI Name | CDP-α-D-glucose(2−)
ChEBI ASCII Name | CDP-alpha-D-glucose(2-)

N clue what this would cost to implement nor whether it would help these users.

**Comment by robnash** [comment]
Hi,
Thanks for adding these terms process terms! I just took a look at the new terms using quickgo and was wondering why there is not a regulates relationship between 'regulation of microlipophagy' (GO:0140505; https://www.ebi.ac.uk/QuickGO/term/GO:0140505) and 'microlipophagy' (GO:0140504; https://www.ebi.ac.uk/QuickGO/term/GO:0140504) by analogy to recent terms I requested 'regulation of UDP-N-acetylglucosamine biosynthetic process' (GO:0106278; https://www.ebi.ac.uk/QuickGO/term/GO:0106278 ) and 'UDP-N-acetylglucosamine biosynthetic process' (GO:0006048; https://www.ebi.ac.uk/QuickGO/term/GO:0006048)? Or maybe I am missing something?

Cheers,
Rob

**Comment by RLovering** [comment]
Please could you suggest the start and end of this process. The decision about whether to curate a protein as involved in the process or being upstream of the process is very hard to make and without a decision on the start and end errors in annotation often arise. For example is the start of the process the recognition of the lipid droplets, so that the proteins detecting the lipid would be annotated to this term. And does the process end when the lysosome-like vacuole is formed or when the vacuole is empty?

Thanks

Ruth

**Comment by robnash** [comment]
Not 100% sure exactly what constitutes the start and end of this process. Here is what I read. According to Seo et al., 2017 (283942507) the initiation of microautophagy is defined as the point where liquid ordered microdomains are formed at sites of engulfment and this requires Atg32p and Atg21p (reviewed in PMID: 29601311), as well as Niemann-Pick type C (NPC) sterol transporter proteins, Ncr1p and Npc2p (reviewed in PMID:29293450).  Then there is a redistribution of Atg14p from ER exit sites onto liquid-ordered vacuole membrane domains through interaction with stabilized AMP-activated protein kinase (AMPK), and together with Atg6p facilities docking and internalization of lipid droplets (LDs) at sites of invagination. It is not clear, at least to me what constitutes the end of this process, but it must at least involve the degradation of LDs for anabolism

**Comment by pgaudet** [comment]
Hi @robnash @RLovering 
I added the missing logical definition and expanded the definition to add the additional information you have in hte previous comment: "Microautophagy-mediated direct internalization of lipid droplets into a lysosome-like vacuole during nutrient depletion, such as during the transition to stationary phase or in response to nutrient limitation. Microlipophagy is mediated by the formation of sterol-enriched vacuolar microdomains at sites of engulfment. Initiation of microautophagy is defined as the point where liquid-ordered microdomains are formed at sites of engulfment, that requires Atg32p and Atg21p, as well as Niemann-Pick type C (NPC) sterol transporter proteins, Ncr1p and Npc2p. This is followed by redistribution of Atg14p from ER exit sites onto liquid-ordered vacuole membrane domains through interaction with stabilized AMP-activated protein kinase (AMPK), and together with Atg6p facilities docking and internalization of lipid droplets (LDs) at sites of invagination." 


I noticed that some papers talk about 'macrolipophagy', for example PMC5836320, section 3.1. Should I also add this term ? Or is it a synonym to an existing term ? 

Thanks, Pascale
